src/app/util/form-controls.ts

```diff
@@ -0,0 +1,26 @@
+import { FormControl, Validators } from '@angular/forms';
+
+export const telefoneFormControl = new FormControl('', [
+    Validators.required
+]);
+export const cpfFormControl = new FormControl('', [
+    Validators.required
+]);
+export const nomeFormControl = new FormControl('', [
+    Validators.required,
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6334730)